core/src/components/badge/badge.ionic.scss

```diff
@@ -174,3 +175,32 @@
   min-width: globals.$ion-scale-400;
   height: globals.$ion-scale-400;
 }
+
+// Badge Inside Tab Button
+// --------------------------------------------------
+
+:host([vertical].in-tab-button) {
+  position: relative;
+
+  min-width: badge-ionic-vars.$badge-ionic-in-tab-button-min-width;
+}
+
+// Badge Icon when Inside Tab Button
+:host([vertical].in-tab-button) ::slotted(ion-icon) {
+  @include globals.position($top: 50%, $start: 50%);
+
+  position: absolute;
+
+  transform: translate(-50%, -50%);
+}
+
+// Badge Icon sizes when inside Tab Button
+:host([vertical].in-tab-button.badge-xxsmall) ::slotted(ion-icon),
+:host([vertical].in-tab-button.badge-xsmall) ::slotted(ion-icon),
+:host([vertical].in-tab-button.badge-small) ::slotted(ion-icon),
+:host([vertical].in-tab-button.badge-medium) ::slotted(ion-icon),
+:host([vertical].in-tab-button.badge-large) ::slotted(ion-icon),
+:host([vertical].in-tab-button.badge-xlarge) ::slotted(ion-icon) {
+  width: globals.$ion-scale-300;
+  height: globals.$ion-scale-300;
+}
```

Comment:
This isn't needed. I was able to remove it on my end and verified that the sizes are still correct since we are using the correct size="xxsmall" prop.

```suggestion
```